writingIdomatic.py

```diff
@@ -17,14 +17,15 @@
 # Notes:    Don’t repeat a '==' comparison in an 'if' statement, use a list or set when you can. ‘x’ in ‘some_list’ can return true or false
 # Ex:
 
+NAME = 'Thomas'
+
 is_generic_name = False
-name = 'Thomas'
-if name == 'Thomas' or name == 'Amir' or name == 'Brett':
+if NAME == 'Thomas' or NAME == 'Amir' or NAME == 'Brett':
```

Comment:
It should be:

THOMAS = 'Thomas'
AMIR = 'Amir'
BRETT = 'Brett'

if name == THOMAS or name == AMIR or name == BRETT

Reply:
Makes sense for the constant names
For the underscore, work has told me to use a leading undersore for variables that aren't used. 
An example is the os.popen which returns console height and width but you might only use one .

Reply:
That... is a weird use. They don't use underscores for private variables?

Reply:
https://www.python.org/dev/peps/pep-0008/#constants

We use PEP8 and from what I can tell that formatting style for constants is just all caps with underscores

Reply:
Ok that's fine. Also, "Use one leading underscore only for non-public methods and instance variables".

That should clear up the confusion about using underscores for instance variables.

Reply:
Ah yes that makes sense. Thanks for pointing that out.